dist-persist/wbstack/src/Internal/ApiWbStackElasticSearchInit.php

```diff
@@ -27,7 +30,13 @@ public function execute() {
 		];
 		$this->getResult()->addValue( null, $this->getModuleName(), $res );
     }
+
     public function getAllowedParams() {
-        return [];
+        return [
+            'cluster' => [
+                ParamValidator::PARAM_TYPE => 'string',
```

Comment:
suggestion: might be worth adding a comment (or even https://doc.wikimedia.org/mediawiki-core/master/php/classWikimedia_1_1ParamValidator_1_1ParamValidator.html#acd1d7a4f0976c7cfc3edc17bdd81c594 ? ) to hint to future us that this is a `cluster` as defined by the cirrus maintenance script classes.